Source/JavaScriptCore/dfg/DFGFixupPhase.cpp

```diff
@@ -1186,6 +1186,10 @@ class FixupPhase : public Phase {
             break;
         }
 
+        case GetByValWithThis: {
+            break;
```

Comment:
Should we predict something here?

Reply:
In the future, yes. Currently, we are not doing any optimizations with speculations.